src/backend/marsha/core/models/account.py

```diff
@@ -226,6 +226,15 @@ class ConsumerSite(BaseModel):
         blank=True,
     )
 
+    show_download = models.BooleanField(
```

Comment:
should we convey in the name of the field that the idea is to configure the DEFAULT behavior?
show_download_default?

Reply:
I don't know, we should maybe add in a name of the field `video`, `show_video_download` as it is related to video's objects. But for default, I don't know

Reply:
You're right! `video_show_download_default`?

Reply:
Done !